templates/coredns/README.md

```diff
@@ -0,0 +1,93 @@
+## template: github.com/threefoldtech/0-templates/coredns/0.0.1
+
+### Description:
+This is a node template responsible for managing [coredns](https://coredns.io/) server instance.
+
+### Schema:
+- `etcdServerName`: the name of etcd server that's created to get ip of it
```

Comment:
`etcd` is enough
and the documentatio should say: `instance name of the etcd service used for configuration`